backtrace-library/src/main/java/backtraceio/library/base/BacktraceBase.java

```diff
@@ -514,4 +542,29 @@ private void addReportAttachments(BacktraceReport report) {
             }
         }
     }
+
+    /**
+     * Custom callback to be executed on server response to a unique events submission request
+     *
+     * @param callback object with method which will be executed
+     */
+    public void setUniqueEventsOnServerResponse(EventsOnServerResponseEventListener callback) {
+        backtraceApi.setUniqueEventsOnServerResponse(callback);
+    }
+
+    /**
+     * Custom callback to be executed on server response to a summed events submission request
+     *
+     * @param callback object with method which will be executed
+     */
+    public void setSummedEventsOnServerResponse(EventsOnServerResponseEventListener callback) {
+        backtraceApi.setSummedEventsOnServerResponse(callback);
+    }
+
+    /**
+     * Get timestamp in seconds
+     */
+    public static long getTimestampSeconds() {
```

Comment:
why `getTimestampSeconds` belong to BacktraceClient domain? wouldn't it be better if this will be available in a helper implementation?

Reply:
Ah okay makes sense, will add like `BacktraceTimeHelper` or something